library/src/test/java/org/apache/apex/malhar/lib/state/managed/DefaultBucketTest.java

```diff
@@ -227,4 +233,33 @@ public void testFreeMemory() throws IOException
 
     testMeta.defaultBucket.teardown();
   }
+
+  @Test
+  public void testBloomFilter() throws IOException
```

Comment:
Yes, bloom filter should only helpful to increase the performance which should be tested in Benchmark application as it need run for a long time . I think what we should test in unit test is apply bloom filter will not break the current result.